.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
I think this should be in the Windows PR?

Reply:
I've then removed it, I'll add it into Windows PR.
Thank you for your review 👍